setup.py

```diff
@@ -28,7 +28,7 @@
 version of Python. If you can't upgrade your pip (or Python), request
 an older version of Django:
 
-    $ python -m pip install "django<2"
+    $ python -m pip install "django<4"
```

Comment:
1.11 was the last version compatible with Python 2 though no?

Reply:
Yes, but as `REQUIRED_PYTHON` is 3.8, if `CURRENT_PYTHON` were 3.7, this would trigger. You'd be told to install `django<2` when higher versions are supported by Python 3.7.